app/utils/install-crisp.js

```diff
@@ -1,21 +0,0 @@
-// jshint ignore: start
-import ENV from 'irene/config/environment';
-
-const installCrisp = function () {
-  if (!ENV.crispWebsiteId || ENV.crispWebsiteId === '') {
-    // eslint-disable-next-line no-console
-    return console.log("Crisp Disabled");
-  }
-  window.$crisp = [];
-  window.$crisp.push(['do', 'chat:hide']);
```

Comment:
Can you verify locally if chat container is hidden by default?

Reply:
default is hidden